openhgnn/models/MAGNN.py

```diff
@@ -425,6 +466,8 @@ def mp_instance_sampler(g, metapath_list):
         the dgl heterogeneous graph
     metapath_list : list
         the list of metapaths in g
+    dataset : str
+        the name of dataset, e.g 'imdb4MAGNN'
```

Comment:
Looks like this is only for generating the cache files.  I guess I will probably replace this with the actual path that stores the files and .  Sometimes `openhgnn/output/MAGNN` may not exist, or the current directory may not have enough storage, etc.

Reply:
Same as the review above.